website/views/pages/try-fleet/register.ejs

```diff
@@ -0,0 +1,42 @@
+<div id="register" v-cloak>
+  <div style="max-width:  750px; padding-top: 80px; padding-bottom: 80px;" class="container d-flex flex-md-row flex-column justify-content-center align-items-center mx-auto px-4 px-md-3">
+    <div>
+    <img purpose="sandbox-image" style="width: 300px; height: auto;" src="/images/fleet-sandbox-300x244@2x.png" alt="Play in Fleet sandbox">
+    </div>
+    <div style="max-width: 400px;" class="flex-column d-flex">
+      <h2>Play in Fleet Sandbox</h2>
+      <p>
+        Fleet Sandbox is designed for testing Fleet features only.<br>
+        <a href="/docs/deploying">Click here</a> for production-ready deployments.
+      </p>
+      <div class="pt-4">
+        <ajax-form :handle-submitting="handleSubmittingRegisterForm" :syncing.sync="syncing" :cloud-error.sync="cloudError" :form-errors.sync="formErrors" :form-data="formData" :form-rules="formRules" @submitted="submittedRegisterForm()">
+          <div class="form-group mb-3">
+            <input id="emailAddress" type="text" class="form-control d-flex w-100" :class="[formErrors.emailAddress ? 'is-invalid' : '']" v-model.trim="formData.emailAddress" placeholder="Sign up with email">
+            <div class="invalid-feedback mt-2">This doesn’t appear to be a valid email address</div>
+          </div>
+          <div class="form-group mb-3">
+            <input id="password" type="password" class="form-control d-flex w-100" :class="[formErrors.password ? 'is-invalid' : '']" v-model.trim="formData.password" placeholder="Choose a password">
+            <div class="invalid-feedback mt-2" v-if="formErrors.password === 'minLength'">Password too short.</div>
+            <div class="invalid-feedback mt-2" v-if="formErrors.password === 'custom'">Passwords must be at least 12 characters long, and must contain at least one letter, number and special character (!@#$%^&*).</div>
```

Comment:
If you get rid of the validation in the other places, I'd suggest removing it here, even though painful.

Also, the & should be &amp; so that it is an encoded HTML entity. It might technically work as-is, but the red highlight you see is because ampersands are supposed to be escaped as &amp; (aka an HTML entity) in HTML text nodes (aka loose text that lives between <foo></foo>)